sdk/servicebus/service-bus/test/utils/testutils2.ts

```diff
@@ -183,10 +183,15 @@ export class ServiceBusTestHelpers {
           sessionId: id
         });
         const msgs = await receiverClient.receiveBatch(numOfMsgsWithSessionId[id], {
-          // To Do - Maybe change the maxWaitTime
-          // Currently set same as numberOfMessages being received
-          maxWaitTimeSeconds: numOfMsgsWithSessionId[id]
+          // Since we know the exact number of messages to be received per session-id,
+          //   `maxWaitTimeSeconds` being a higher number is not a problem
+          maxWaitTimeSeconds: 5 * numOfMsgsWithSessionId[id]
```

Comment:
Just this change seems to fix things..
Before - https://dev.azure.com/azure-sdk/internal/_build/results?buildId=331350&view=logs&j=291aad5e-8689-5983-e460-98c01ad4e5a9
After - https://dev.azure.com/azure-sdk/internal/_build/results?buildId=331465&view=logs&j=291aad5e-8689-5983-e460-98c01ad4e5a9